.changeset/vast-roses-remain.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/factory': minor
+---
+
+Added Slack channel adapter options to SlackIntegration and made concise thinking, typing, and working statuses the default.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)